homeworks/homework_1/task_2/wc.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import os
+from typing import List
+
+
+def print_wordstats(file_names: List[str]):
+    total_lines = 0
+    total_words = 0
+    total_bytes = 0
+
+    for file_name in file_names:
+        try:
+            with open(file_name) as f:
+                file_text = f.readlines()
```

Comment:
`readlines()` is very memory-consuming. You are basically loading the entire file into memory, what if it is 10G? For the purpose of this homework, it is fine for `tails` (it is tricky to do it differently), but for all other tasks, let's do it iteratively, but iterating over file. 

P. S. doing `list(f)` is the same as doing `readlines()` in this regard.